forage-android/src/main/java/com/joinforage/forage/android/LDManager.kt

```diff
@@ -80,4 +99,32 @@ internal object LDManager {
         internalLogger.i("[LaunchDarkly] Vault type set to $vaultType")
         return vaultType as String
     }
+
+    internal fun getPollingIntervals(app: Application, logger: Log, dataSource: TestData? = null): List<Int> {
+        if (pollingIntervals != null) {
+            return pollingIntervals as List<Int>
+        }
+        internalLogger = logger
+        // Datasource is required for testing purposes!
+        val ldConfig = if (dataSource != null) {
+            LDConfig.Builder()
+                .mobileKey(LD_MOBILE_KEY)
+                .dataSource(dataSource)
+                .build()
+        } else {
+            LDConfig.Builder()
+                .mobileKey(LD_MOBILE_KEY)
+                .build()
+        }
+        val contextKind = ContextKind.of(LDContextKind.SERVICE)
+        val context = LDContext.create(contextKind, LDContexts.ANDROID_CONTEXT)
+        val client = LDClient.init(app, ldConfig, context, 0)
+
+        val jsonIntervals = client.jsonValueVariation(LDFlags.ISO_POLLING_WAIT_INTERVALS, LDValue.buildObject().put("intervals", LDValue.buildArray().add(1000).build()).build())
```

Comment:
[LD](https://docs.launchdarkly.com/sdk/client-side/android/migration-2-to-3/?q=jsonvaluevariation#understanding-changes-to-the-variation-methods) introduced `LDValue` in the `jsonValueVariation` method.

We need to determine how to use `LDValue` properly here.